lessons/taquito/06/06.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Originating, finally.
+chapter: Chapter 6
+slug: module-04/chapter-06
+filterBy: lesson-4
+editor:
+  language: javascript
+  startingCode: |
+    const FAUCET_KEY = {
+        "mnemonic": [
+            "chimney",
+            "item",
+            "course",
+            "pair",
+            "opinion",
+            "mass",
+            "receive",
+            "wrong",
+            "hurry",
+            "nest",
+            "oil",
+            "acoustic",
+            "section",
+            "salad",
+            "electric"
+        ],
+        "secret": "47184b1d8bf4dd550256817c3507cc4ccf112888",
+        "amount": "903205016",
+        "pkh": "tz1Q654L2qnt5mDM1Lbi4rMSTtnE5JHvD8M7",
+        "password": "9U5VOADM6l",
+        "email": "ryqwtuyj.ciwjnvdt@tezos.example.org"
+    }
+    
+    importKey(
+        Tezos,
+        FAUCET_KEY.email,
+        FAUCET_KEY.password,
+        FAUCET_KEY.mnemonic.join(' '),
+        FAUCET_KEY.secret
+    );
+    
+    Tezos.setProvider({ rpc: 'https://carthagenet.SmartPy.io' });
+    
+    // const CODE_JSON = 
+    
+    // const STORAGE_JSON = 
+    
+    // Originate the contract -
+    
+
+  answer: |
+    const FAUCET_KEY = {
+        "mnemonic": [
+            "chimney",
+            "item",
+            "course",
+            "pair",
+            "opinion",
+            "mass",
+            "receive",
+            "wrong",
+            "hurry",
+            "nest",
+            "oil",
+            "acoustic",
+            "section",
+            "salad",
+            "electric"
+        ],
+        "secret": "47184b1d8bf4dd550256817c3507cc4ccf112888",
+        "amount": "903205016",
+        "pkh": "tz1Q654L2qnt5mDM1Lbi4rMSTtnE5JHvD8M7",
+        "password": "9U5VOADM6l",
+        "email": "ryqwtuyj.ciwjnvdt@tezos.example.org"
+    }
+    
+    importKey(
+        Tezos,
+        FAUCET_KEY.email,
+        FAUCET_KEY.password,
+        FAUCET_KEY.mnemonic.join(' '),
+        FAUCET_KEY.secret
+    );
+    
+    Tezos.setProvider({ rpc: 'https://carthagenet.SmartPy.io' });
+    
+    // const CODE_JSON = 
+    
+    // const STORAGE_JSON = 
+    
+    // Originate the contract - 
+    Tezos.contract.originate({
+        code: CODE_JSON,
```

Comment:
Add json code preloaded here would resolve the issue. If I run it now, it doesn't throw any error nor display any output since CODE_JSON and STORAGE_JSON is empty.

Reply:
Also, we would need to figure out how to add a peice of code that will tell us that code was originated from cryptocodeschool

Reply:
First, yes I've pre-loaded both the code and storage now.
Second, can we ask somebody about this "Also, we would need to figure out how to add a piece of code that will tell us that code was originated from cryptocodeschool"?